### DISCUSSION TITLE
Filtering old/new records by ‘Date/Time’ filed

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#1910] [Help] [answered]
Hi all. Is it possible to filter by "Created At" filed a rows which are older/never than `x` days from the current date? 
For example, I'd like to get a records which were created more than 60 days ago.
Thanks in advance 🙏

**Comment by sergey-dme** [comment, marked as answer]
I think it might be possible using `subtract` or `add` and` {{ date now }}` but cant figure out the final formula...

**Reply by mike12345567** [reply]
Hi @sergey-dme - you could try something along the lines of `{{ date (subtract (date now "x") 5184000000) "" }}` - the goal here is to take now, translate it to a JS timestamp (seconds since epoch) minus 60 days in milliseconds (60 * 86400000) and then translate it back to an ISO timestamp that the system can use for searching - you could also do the math in there with a `(multiply 60 * 86400000)` instead of `5184000000` if you wanted it to be more obvious/easy to change or reuse.

**Reply by sergey-dme** [reply]
Awesome! That works!  the final formula is: `{{ date (subtract (date now "x")  (multiply 60 86400000)) "" }}` where 60 is number of days. @mike12345567 Thank you very much! budibase is the best platform!

**Reply by twilson9034** [reply]
Did this give you an actual date or just a number timestamp that needs to be converted again?

**Reply by twilson9034** [reply]
I got it

**Reply by silvanhornstein** [reply]
Can u probably provide more details? Using the formula u provided within an automation to filter rows while querying does not work for me.
